### PR TITLE
WMS: 11453 make Tasks heading 2 in Lab 4-5

### PR DESCRIPTION
**Pull request by OraGitPush1**
WMS: 11453 
make Tasks heading 2 in Lab 4-5

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [X] All filenames are lower case (including folders, images, files, etc.)
- [X] Filenames are descriptive
- [X] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [X] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [X] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by OraGitPush1** [comment]
@anooshapilli  #378 added to make Tasks heading 2 in Lab 4-5.

File: sharding/uds23c/topology/ds-topology.md
Comment (denisemyrick) [inline sharding/uds23c/topology/ds-topology.md:1]: Restart numbering in each tasks.

File: sharding/uds23c/uds-queries/uds-queries.md
Comment (denisemyrick) [inline sharding/uds23c/uds-queries/uds-queries.md:1]: Restart numbering in each tasks. 
Task 5 Connect should be capitalized

**Review by denisemyrick — DISMISSED** [review]
- Please update uds-podman-GSM1.png to be all lowercase. 
- Please resolve conflicts
-

**Comment by anooshapilli** [comment]
@OraGitPush1 - Please resolve the conflicting files issue

File: sharding/uds23c/cleanup/ds-cleanup.md
[the remark this answers is not in the document]
Comment (OraGitPush1) [reply sharding/uds23c/cleanup/ds-cleanup.md:1]: done

File: sharding/uds23c/topology/ds-topology.md
Comment (OraGitPush1) [reply sharding/uds23c/topology/ds-topology.md:1]: done

**Review by OraGitPush1 — COMMENTED** [review]
WMS: 11453  ready for review, pull request #378

**Comment by OraGitPush1** [comment]
WMS: 11453 Pull request https://github.com/oracle-livelabs/database/pull/378 read for review

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**